### PR TITLE
Update to the documentation regarding client languages + mention of the CLI

### PR DESCRIPTION
**Pull request by ChristopherAndre**
Following discussion on Dec 17th with @viktorklang during the status meeting call, added list of known client libraries with links to these libraries and basic support matrix.
Also, mentioned the existence of the CLI for generating new projects.

### CODE REVIEW

File: docs/src/main/paradox/user/lang/index.md [hunk 1]
```diff
@@ -11,3 +12,30 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
...
+| Library Name | Language | Event-based support? | CRDT support? |
+|--------------|----------|----------------------|---------------|
+| [scala-support](https://github.com/cloudstateio/cloudstate/tree/master/scala-support/src/main) | Scala | :warning: | :warning: |
+| [kotlin-support](https://github.com/cloudstateio/kotlin-support) | Kotlin | :warning: | :warning: |
+| [cloudstate-csharp](https://github.com/nagytech/cloudstate-csharp)| C# | :heavy_check_mark: | :heavy_check_mark: | 
+| [python-support](https://github.com/marcellanz/cloudstate_python-support/tree/feature/python-support)| Python | | |
+| | Elixir | | |
+| | F# | | |
+| | Ruby | | |
+| | Rust | | |
```
Comment (sleipnir) [inline docs/src/main/paradox/user/lang/index.md]: follow repository link https://github.com/sleipnir/cloudstate-rust

File: docs/src/main/paradox/user/lang/index.md [hunk 2]
```diff
@@ -11,3 +12,30 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
...
+ 
+Here is a list of other, **non-official** libraries in various other languages:
+
+| Library Name | Language | Event-based support? | CRDT support? |
+|--------------|----------|----------------------|---------------|
+| [scala-support](https://github.com/cloudstateio/cloudstate/tree/master/scala-support/src/main) | Scala | :warning: | :warning: |
+| [kotlin-support](https://github.com/cloudstateio/kotlin-support) | Kotlin | :warning: | :warning: |
+| [cloudstate-csharp](https://github.com/nagytech/cloudstate-csharp)| C# | :heavy_check_mark: | :heavy_check_mark: | 
+| [python-support](https://github.com/marcellanz/cloudstate_python-support/tree/feature/python-support)| Python | | |
+| | Elixir | | |
```
Comment (sleipnir) [inline docs/src/main/paradox/user/lang/index.md]: You may want to remove implementations that do not currently have a repository like Elixir, F#, Ruby, etc

File: docs/src/main/paradox/user/lang/index.md [hunk 3]
```diff
@@ -25,16 +25,14 @@ Here is a list of other, **non-official** libraries in various other languages:
 | [scala-support](https://github.com/cloudstateio/cloudstate/tree/master/scala-support/src/main) | Scala | :warning: | :warning: |
 | [kotlin-support](https://github.com/cloudstateio/kotlin-support) | Kotlin | :warning: | :warning: |
 | [cloudstate-csharp](https://github.com/nagytech/cloudstate-csharp)| C# | :heavy_check_mark: | :heavy_check_mark: | 
-| [python-support](https://github.com/marcellanz/cloudstate_python-support/tree/feature/python-support)| Python | | |
-| | Elixir | | |
-| | F# | | |
-| | Ruby | | |
-| | Rust | | |
```
Comment (sleipnir) [inline docs/src/main/paradox/user/lang/index.md]: Rust already has a repository that you can use if you wish: https://github.com/sleipnir/cloudstate-rust

File: docs/src/main/paradox/user/lang/index.md [hunk 4]
```diff
@@ -11,3 +12,28 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
 * [Go](go/index.md)
 
 @@@
+
+## Unofficial client libraries
+We are aware of external libraries implementing the **client CloudState protocol** in various other languages.
```
Comment (marcellanz) [inline docs/src/main/paradox/user/lang/index.md]: @sleipnir Cloudstate was recently renamed by lightbend from Cloud**S**tate to Cloud**s**tate; hence mind the s that went from uppercase to lowercase.

File: docs/src/main/paradox/user/lang/index.md [hunk 5]
```diff
@@ -11,3 +12,28 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
...
+
+:warning: **Though they are mentioned here, these libraries are often not directly maintained by the CloudState dev team and are therefore not supported**.
```
Comment (marcellanz) [inline docs/src/main/paradox/user/lang/index.md]: CloudState => Cloudstate

File: docs/src/main/paradox/user/lang/index.md (hunk 4 above)
Comment (sleipnir) [reply docs/src/main/paradox/user/lang/index.md]: Thanks Marcel

Comment (marcellanz) [reply docs/src/main/paradox/user/lang/index.md]: @ChristopherAndre … ouch, I'm so sorry, I did not mention you correctly in my last comment. Thanks for the contribution.

Comment (ChristopherAndre) [reply docs/src/main/paradox/user/lang/index.md]: Fixed in the latest version!

File: docs/src/main/paradox/user/lang/index.md [hunk 6]
```diff
@@ -11,3 +12,28 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
...
 @@@
+
+## Unofficial client libraries
+We are aware of external libraries implementing the **client Cloudstate protocol** in various other languages.
+
+:warning: **Though they are mentioned here, these libraries are often not directly maintained by the Cloudstate dev team and are therefore not supported**.
+ 
+Here is a list of other, **non-official** libraries in various other languages:
+
+| Library Name | Language | Event-based support? | CRDT support? |
```
Comment (viktorklang) [inline docs/src/main/paradox/user/lang/index.md]:
```suggestion
| Library Name | Language | Event-sourcing support? | CRDT support? |
```

File: docs/src/main/paradox/user/lang/index.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 # Languages
 
+## Officially supported client libraries
```
Comment (viktorklang) [inline docs/src/main/paradox/user/lang/index.md]: I'm not sure we should divide things into Officially supported and not officially supported—instead I think we should list "Known language supports" and then list supported features and versions.

Comment (ChristopherAndre) [reply docs/src/main/paradox/user/lang/index.md]: Guess I've been working for editors for too long 😄 
What about removing the `Officially supported client libraries` title and renaming the other one to `3rd party libraries`?

Comment (viktorklang) [reply docs/src/main/paradox/user/lang/index.md]: Sounds like a great idea, @ChristopherAndre! Make that change and then I think we are ready to merge

Comment (ChristopherAndre) [reply docs/src/main/paradox/user/lang/index.md]: Done!

**Comment by viktorklang** [comment]
Thanks @ChristopherAndre!

**Pull request merged**